src/main/java/orm/dsl/dml/InsertImpl.java

```diff
@@ -33,35 +36,64 @@ public InsertImpl(TableEntity<E> tableEntity, QueryRunner queryRunner) {
     }
 
     @Override
-    public <T> InsertIntoStep values(T entity) {
+    public InsertIntoValuesStep<E> value(E entity) {
         throwIfNotMatchingEntity(tableEntity, entity);
         this.inertValues = List.of(extractInsertValues(entity));
         return this;
     }
 
+    /**
+     * Bulk Insert는 values() 중복실행 가능
+     * @param entityList insert 할 엔티티 리스트
+     * @param <T> 엔티티
+     */
     @Override
-    public <T> InsertIntoStep values(List<T> entityList) {
+    public <T> BulkInsertIntoValuesStep values(List<T> entityList) {
         for (T entity : entityList) {
             throwIfNotMatchingEntity(tableEntity, entity);
         }
 
-        this.inertValues = entityList.stream()
+        final List<List<? extends TableField>> newEntityList = entityList.stream()
                 .map(this::extractInsertValues)
-                .collect(Collectors.toUnmodifiableList());
+                .collect(Collectors.toList());
+
+        if (CollectionUtils.isEmpty(newEntityList)) {
+            this.inertValues = newEntityList;
+            return this;
+        }
+
+        if(this.inertValues == null) {
+            this.inertValues = new ArrayList<>();
+        }
 
+        this.inertValues.addAll(newEntityList);
         return this;
     }
 
+    /**
+     * 입력한 값을 Auto Increment가 존재하면 포함해서 리턴하는 기능
+     * @return
```

Comment:
```suggestion
     * @return E
```
주석을 다는 만큼, 명확한 정보를 전달하기 위해 신경써주세요~ 😄 
저같은 경우는 주석이 있으면, 자연스럽게 주석으로 시선이 먼저가더라구요,
그런데 그것이 구현이랑 다른것 같으면 코드의 신뢰성이 떨어지지 않을까 생각되어요 🤔

Reply:
넵 이부분은 신경쓰도록 하겠습니다 :)